src/views/v2/month/calendar-event/tooltip.php

```diff
@@ -12,11 +12,17 @@
  * @version TBD
  *
  */
-// $event    = $this->get( 'event' );
-// $event_id = $event->ID;
+$event    = $this->get( 'event' );
+$event_id = $event->ID;
 ?>
-<div class="tribe-events-calendar-month__calendar-event-tooltip">
-	<?php $this->template( 'month/calendar-event/tooltip/featured-image', [ 'event' => $event ] ); ?>
-	<?php $this->template( 'month/calendar-event/tooltip/description', [ 'event' => $event ] ); ?>
-	<?php /* RSVP Ticket CTA goes here */ ?>
+<div class="tribe-events-tooltip__content">
```

Comment:
no need for wrapper class, add `data-js` attribute to div below.

Reply:
Yes, unfortunately for the a11y implementation we need a wrapper.